vehicle/accel_brake_map_calibrator/accel_brake_map_calibrator/scripts/new_accel_brake_map_server.py

```diff
@@ -0,0 +1,339 @@
+
+        # remove unused_data
+        self.csv_data = self.cr.removeUnusedData(
+            self.min_vel_thr,
+            self.max_steer_thr,
+            self.max_pitch_thr,
+            self.max_pedal_vel_thr,
+            self.max_jerk_thr,
+        )
+
+        # get statistics array
+        vel_data = self.cr.getVelData()
+        pedal_data = self.cr.getPedalData()
+        acc_data = self.cr.getAccData()
+
+        # get color factor (pitch) array for plotting
+        color_data = self.cr.getPitchData()
+
+        data, full_data = CalcUtils.create_2d_map(
+            vel_data,
+            pedal_data,
+            acc_data,
+            color_data,
+            CF.VEL_LIST / 3.6,
+            self.vel_diff_thr,
+            CF.PEDAL_LIST,
+            self.pedal_diff_thr,
+        )
+
+        count_map, average_map, stddev_map = CalcUtils.create_stat_map(data)
+        velocity_map_list = []
+        for i in range(len(CF.VEL_LIST)):
+            velocity_map_list.append(CalcUtils.extract_x_index_map(full_data, i))
+
+        default_pedal_list, default_acc_list = self.load_map(self.default_map_dir)
+        if len(default_pedal_list) == 0 or len(default_acc_list) == 0:
+            self.get_logger().warning(
+                "No default map file was found in {}".format(self.default_map_dir)
+            )
+
+            response.graph_image = []
+            print("svg data is empty")
+
+            response.accel_map = ""
+            print("accel map is empty")
+
+            response.brake_map = ""
+            print("brake map is empty")
+
+            return response
+
+        calibrated_pedal_list, calibrated_acc_list = self.load_map(self.calibrated_map_dir)
+        if len(calibrated_pedal_list) == 0 or len(calibrated_acc_list) == 0:
+            self.get_logger().warning(
+                "No calibrated map file was found in {}".format(self.calibrated_map_dir)
+            )
+
+        # visualize point from data
+        plot_width = 3
+        plot_height = int(math.ceil(len(CF.VEL_LIST) / float(plot_width)))
+        plotter = Plotter(plot_height, plot_width)
+        for i in range(len(CF.VEL_LIST)):
+            self.view_pedal_accel_graph(
+                plotter,
+                i,
+                velocity_map_list,
+                i,
+                count_map,
+                average_map,
+                stddev_map,
+                default_pedal_list,
+                default_acc_list,
+                calibrated_pedal_list,
+                calibrated_acc_list,
+            )
+        plt.savefig("plot.svg")
+        print("svg saved")
+
+        # pack response data
+        text = Path("plot.svg").read_text()
+        if text == "":
+            response.graph_image = []
+            print("svg data is empty")
+        else:
+            byte = text.encode()
+            for b in byte:
+                response.graph_image.append(b)
+            print("svg data is packed")
```

Comment:
Change print to logger info in this file.If test code, delete these print.

`self.get_logger().info()`

Reply:
Changed `print` to `self.get_logger().info()` or `self.get_logger().info().warning()`